modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc

```diff
@@ -53,17 +53,54 @@ spec:
 <1> The index image URL contains the desired Operator images. If the index images are always pushed to the same image name and tag, this change is not needed.
 <2> Set how frequently the Operator Lifecycle Manager (OLM) polls the index image for new Operator versions with the `registryPoll.interval` field. This change is not needed if a new index image tag is always pushed for y-stream and z-stream Operator updates. The `registryPoll.interval` field can be set to a shorter interval to expedite the update, however shorter intervals increase computational load. To counteract this, you can restore `registryPoll.interval` to the default value once the update is complete.
 <3> Last observed state of the catalog connection. The `READY` value ensures that the `CatalogSource` policy is ready, indicating that the index pod is pulled and is running. This way, {cgu-operator} upgrades the Operators based on up-to-date policy compliance states.
++
+[NOTE]
+====
+After a catalog source update, it takes time for OLM to update the subscription status. This can mean that the status of the subscription policy may continue to show as compliant when TALM decides whether remediation is needed. As a result the Operator specified in the subscription policy does not get upgraded. To mitigate this issue, the `lastObservedState: READY` specification reduces the time between the completion of the catalog source policy remediation and the update of the subscription status. 
+
+However, if the issue persists, you can add another catalog source configuration to the `PolicyGenTemplate` resource with the same specifications, except for the following changes: 
```

Comment:
I'm unclear if the "same specifications" indicate as some default settings or something more specific that is described elsewhere?

Reply:
Just wondering if the follow YAML guideline applies here?

![Screenshot from 2023-09-25 16-49-27](https://github.com/openshift/openshift-docs/assets/57954076/4a2b4033-04c6-478b-8846-829e032a4a35)

[Link](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting-for-kubernetes-and-openshift-api-objects)

Reply:
Oh, I wasn't aware of the ellipsis issue. Do you know if this rule is enforced @bergerhoffer ? I got 772 hits for this in the repo...

Reply:
I know we talked about it on slack already, but just posting here too that, yes, we do try to enforce it. But it does get missed a lot, so it's a mixed bag in the repo.